lib/util.js

```diff
@@ -52,7 +52,7 @@ exports.strToChars = function(str) {
 exports.tokenizeClass = (str, regexpStr) => {
   /* jshint maxlen: false */
   var tokens = [];
-  var regexp = /\\(?:(w)|(d)|(s)|(W)|(D)|(S))|((?:(?:\\)(.)|([^\]\\]))-(?:\\)?([^\]]))|(\])|(?:\\)?(.)/g;
+  var regexp = /\\(?:(w)|(d)|(s)|(W)|(D)|(S))|((?:(?:\\)(.)|([^\]\\]))-(?:\\)?([^\]]))|(\])|(?:\\)?(.|[\r\n])/g;
```

Comment:
You can use `([^])`, which will also work for U+2028 and U+2029.

Reply:
Great idea, I will do that.